test/mission-network-isolation.test.js

```diff
@@ -90,12 +104,21 @@ describe('Sensors test', () => {
       notificationCategory: getShortUnique(), //💡 TIP: Unique category will lead to unique notification URL. This helps in overriding the nock
     });
     // 💡 TIP: Set here a nock that replies with 500: nock('http://localhost').post(`/notification/${eventToAdd.notificationCategory}`)
-
+    nock('http://localhost').post(`/notification/${eventToAdd.notificationCategory}`).reply(500, { success: false, });
     // Act
+    const savedEvent = await request(expressApp)
+      .post('/sensor-events')
+      .send(eventToAdd);
 
+    const receivedResponse = await request(expressApp)
```

Comment:
Just move this lines to assert and we are OK! Since this are not part of the action, are just the consequences.